### DISCUSSION TITLE
Select color for a button

### DISCUSSION THREAD

**Discussion by pgamerx** [squidfunk/mkdocs-material#3348] [Q&A] [answered]
Hi there, I am new to MkDocs. 
I was wondering how I can add a different colour for the button i.e red in my case.
Currently my code is:
```md
[Register with Google :material-google:](#){ .md-button }
```
is there any CSS class for red color?

**Comment by squidfunk** [comment, marked as answer]
You can use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css), add a new class and add that to the attribute list as well:

``` css
.md-button--foo {
  background: red;
}
```

``` markdown
[Register with Google :material-google:](#){ .md-button.md-button--foo }
```

**Reply by pgamerx** [reply]
Thank you!
